### PR TITLE
improvement: upgrade @cognite/sdk to 5.x

### PR DESCRIPTION
**Pull request by larsmoa**

### CODE REVIEW

File: documentation/package.json [hunk 1]
```diff
@@ -16,12 +16,16 @@
     "apiref": "yarn run apiref:generate && yarn run apiref:concat",
     "apiref:generate": "typedoc --entryPoints ../viewer/src/index.ts --entryPoints ../viewer/src/tools.ts --listInvalidSymbolLinks",
     "apiref:concat": "concat-md ./generated --startTitleLevelAt 1 --decrease-title-levels --dir-name-as-title > \"./docs/API Reference.md\"",
-    "replace-1.x": "yarn apiref && rimraf versioned_sidebars versioned_docs versions.json && yarn docusaurus docs:version 1.x"
+    "replace-1.x:copy-docs": "yarn apiref && rimraf versioned_sidebars versioned_docs versions.json && yarn docusaurus docs:version 1.x",
+    "replace-1.x:update-sdk": "yarn replace \"\\'@cognite/sdk\\'\" \"''@cognite/sdk-1.x''\" versioned_docs/version-1.x/ --include '*.ts*' -r",
+    "replace-1.x:update-reveal": "yarn replace \"\\'@cognite/reveal\\'\" \"''@cognite/reveal-1.x''\" versioned_docs/version-1.x/ --include '*.ts*' -r",
+    "replace-1.x": "yarn replace-1.x:copy-docs && yarn replace-1.x:update-sdk && yarn replace-1.x:update-reveal"
```
Comment (christjt) [inline documentation/package.json:22]: I have absolutely no idea what is going on there, but if it works it works I guess

File: examples/src/pages/Simple.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ export function Simple() {
         modelUrl: 'primitives',
       });
       const client = new CogniteClient({ appId: 'reveal.example.simple' });
-      client.loginWithOAuth({ project });
+      await client.loginWithOAuth({ type: 'CDF_OAUTH', options: { project }});
```
Comment (christjt) [inline examples/src/pages/Simple.tsx:33]: Why doesn't this need `await client.authenticate();` ?

**Review by christjt — CHANGES_REQUESTED** [review]
I don't really know the implications of switching to the new SDK version, but as long as it works, everything should be fine :)

File: documentation/package.json (hunk 1 above)
Comment (larsmoa) [reply documentation/package.json:22]: This replaces import of Reveal and Cognite SDK with `@cognite/sdk-1.x` and `@cognite/reveal-1.x` to be able to have separate versions for docs for 1.x and 2.x. Before this was done manually.

File: examples/src/pages/Simple.tsx (hunk 2 above)
Comment (larsmoa) [reply examples/src/pages/Simple.tsx:33]: It does :| I'll fix.

**Review by christjt — APPROVED** [review]

**Pull request merged**